lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1357,11 +1360,6 @@ def ticklabel_format(
             sb = True
         elif style in ['plain', 'comma']:
             sb = False
-            if style == 'plain':
```

Comment:
just drop support for 'comma' style (deprecation, yada yada)?  that doesn't exist for regular axes either.